frontend/src/app.vue

```diff
@@ -298,27 +324,7 @@ const app = {
       return REPOS[hashParams.tabRepo].location.location;
```

Comment:
Is it possible to add typing for line 318 as well

Reply:
The `window` object in 318 is typed from `types/window.ts`, so the objects in this function should all be implicitly typed

<details>
<summary>gif</summary>

![app-types](https://user-images.githubusercontent.com/45852430/226164180-c2541eab-d5c9-452f-ac36-bffca497766c.gif)
</details>